src/components/event_forms/styles/EventForm.module.scss

```diff
@@ -159,3 +206,38 @@
     margin-left: $gap-sm;
   }
 }
+
+.cal {
```

Comment:
We will probably want to think about going back through some of the code we've done and add media queries for mobile and tablet devices

Reply:
Media queries added as per suggestion. Please review!